src/utility.cpp

```diff
@@ -98,10 +96,14 @@ namespace azure {  namespace storage_lite {
         }
         else if (format == date_format::rfc_1123)
         {
-            std::stringstream ss;
-            ss.imbue(std::locale("C"));
-            ss << std::put_time(pm, constants::date_format_rfc_1123);
-            return ss.str();
+            const char* weekdays[] = { "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" };
```

Comment:
If the compiler does not optimize this, use static keyword to avoid recreating the arrays please.

Reply:
fixed